src/components/Messages/Message/index.js

```diff
@@ -93,6 +99,7 @@ const renderContent = ({
 		<MessageBubble inverse={me} msgSequence={msgSequence} quoted={quoted} system={system}>
 			{showPostChatUrl && <Button onClick={() => onClickSurvey()} className = {createClassName(styles, 'surveyButton__content')}>Give Feedback</Button>}
 			<MessageText style={showPostChatUrl && { width: '300px' }} text={text} system={system} />
+			{chatIsClosed && <Button onClick={() => onClickTranscript()} className = {createClassName(styles, 'transcriptButton__content')}>Download Transcript</Button>}
```

Comment:
How about `isChatClosed` instead of `chatIsClosed`?

Also I forgot to mention on Aditya's PR, we should use `I18n.t('Download Transcript')` for the text content as best practice